core/profiles/standard/standard.install

```diff
@@ -108,6 +108,42 @@ function standard_install() {
   theme_enable(array($default_theme));
   theme_disable(array('stark'));
 
+  // Enables the title_combo block and positions breadcrumbs
+  // immediately above title_combo block.
+
+  $title_combo_uuid = 'a549d0bc-51f7-46b0-9b41-653788410dd4';
+
+  $title_combo_block = array(
+    'content.' . $title_combo_uuid => array(
+      'plugin' => 'system:page_components:title_combo',
+      'data' => array(
+        'module' => 'system',
+        'delta' => 'page_components',
+        'settings' => array(
+          'title_display' => 'default',
+          'title' => '',
+          'style' => 'default',
+          'block_settings' => [],
+          'contexts' => [],
```

Comment:
These `[]`s will break php5.3 tests (because we're in a php file - not a .json file)

```suggestion
          'block_settings' => array(),
          'contexts' => array(),
```

Reply:
@klonos - Thanks. Done.